src/api/connectors/SpotifyConnector.ts

```diff
@@ -42,6 +44,21 @@ class SpotifyConnector implements ConnectorHandler {
                 payload.meta.episode,
                 payload.data as SpotifyListenersPayload
             )
+        } else if (payload.meta.endpoint === 'aggregate') {
+            //validates the payload and throws an error if it is not valid
+            validateJsonApiPayload(aggregateSchema, payload.data)
+
+            // check if episode id exists in metadata
+            if (payload.meta.episode === undefined) {
+                throw new PayloadError('missing episode id')
+            }
+
+            return await this.repo.storeAggregate(
+                accountId,
+                payload.meta.episode,
+                payload.range.start,
```

Comment:
Yeah, start and end is actually 7 days by default. Need to add support for the range parameter in the python library first.

Reply:
https://github.com/openpodcast/api/pull/8

Reply:
or change to 1 day default, start === end would be correct

Reply:
Added the range check. I would still allow payloads where end-start is more than one day because technically it isn't incorrect and we might have a use-case for that in the future.

Reply:
Removed the range check as discussed.